carousel/src/carousel/editor.css

```
@@ -0,0 +1,78 @@
+
+		&:hover {
+			background-color: var(--wp-components-color-accent, var(--wp-admin-theme-color, #3858e9)) !important;
+			color: #fff;
+		}
+	}
+
+	.wp-block-gallery.wp-block-wpcomsp-carousel-track.has-nested-images figure.wp-block-image {
+		flex-direction: unset;
+	}
+
+	.wp-block-wpcomsp-carousel-track:not(.has-child-selected):not(.is-selected) {
```

Stylelint (17.3.0)

[error] Expected complex :not() pseudo-class notation (selector-not-notation)

(selector-not-notation)